hack/ci/e2e-conformance.sh

```diff
@@ -63,9 +74,10 @@ init_image() {
           rm packer.zip && \
           mv packer "$(go env GOPATH)/bin"
       fi
-      (cd "$(go env GOPATH)/src/sigs.k8s.io/image-builder/images/capi" && \
-      GCP_PROJECT_ID=$GCP_PROJECT GOOGLE_APPLICATION_CREDENTIALS=$GOOGLE_APPLICATION_CREDENTIALS \
-      make build-gce-default)
+      # FIXME : seems to be blowing up in the CI.
```

Comment:
Maybe we can have another job that publishes the images in the correct project?

Reply:
y probably!